rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/fragment/CovidPatientRegisterFragment.java

```diff
@@ -53,5 +58,7 @@ public void onSyncComplete(FetchStatus fetchStatus) {
                 .getContext()
                 .allSharedPreferences()
                 .savePreference(Constants.Preference.CTS_LATEST_SYNC_TIMESTAMP, String.valueOf(System.currentTimeMillis()));
```

Comment:
I don't think we'll need to save the timestamp now. Once the broadcast is received, we can use the current time as the timestamp.

Reply:
I think we will have to save the timestamp like when syncing in progress then we will show up the last saved sync timestamp until the sync is not completed.

Reply:
yes, persistence removed.